razorpay/controllers/front/validation.php

```diff
@@ -60,6 +60,32 @@ public function postProcess()
         {
             $payment = $api->payment->fetch($paymentId);
 
+            //validate Rzp signature
+            try
+            {
+                session_start();
+                $attributes = array(
+                    'razorpay_order_id' => $_SESSION['rzp_order_id'],
+                    'razorpay_payment_id' => $_REQUEST['razorpay_payment_id'],
+                    'razorpay_signature' => $_POST['razorpay_signature']
```

Comment:
We're using $_REQUEST for payment id and $_POST for signature. Any reason for this?

Reply:
We are sending in payment id in query string so just used in $_REQUEST. signature we added in form hidden field so $_POST.